Net/include/Poco/Net/SocketImpl.h

```diff
@@ -182,6 +182,16 @@ class Net_API SocketImpl: public Poco::RefCountedObject
 		///
 		/// Always returns zero for platforms where not implemented.
 
+	virtual void sendFile(const std::string& filePath, const size_t fileSize);
+        /// Implemented only on Linux. Noop on other platforms.
```

Comment:
tab indentation, not spaces

Reply:
The last commit screwed it up, I think. I'll fix it tomorrow.

Reply:
Resolved